src/components/DetailsContainer/DetailsContainer.js

```diff
@@ -5,92 +5,90 @@ import Header from '../Header/Header';
 import colorado from '../../assets/colorado-flag.webp';
 import { getSingleTree } from '../../apiCalls';
 
-
-const DetailsContainer = ({selectedTree, clearSelectedTree, clearZipTrees, match}) => {
+const DetailsContainer = ({clearSelectedTree, clearZipTrees, match}) => {
   window.scroll(0,0);
-  console.log('match line 11', match)
-    const [myTree , setMyTree] = useState({})
+  
+  const [myTree , setMyTree] = useState({})
 
-    useEffect(() => {
-      console.log('match',match)
-      getSingleTree(match)
-        .then(data => {
-          setMyTree(data[0])
-        })
-        console.log('myTree',myTree)
-    },[])
+  useEffect(() => {
```

Comment:
Excellent job getting the fetch running on page load, no more undefined data on refresh! 🥳